### PR TITLE
OADP-3710: Fix pod restore plugin log

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: velero-plugins/pod/restore.go [hunk 1]
```diff
@@ -163,7 +163,7 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	// For backups made with OADP 1.3 or later, base this on the presence of any volumes to back up or restore hooks
 	if pod.Annotations != nil && len(pod.Annotations[common.DCIncludesDMFix]) > 0 {
 		disconnectIfDC = podHasRestoreHooks || podHasVolumesToBackUp
-	// For backups made with OADP 1.2 or earlier, use only the defaultVolumesToRestic flag
+		// For backups made with OADP 1.2 or earlier, use only the defaultVolumesToRestic flag
```
Comment (mateusoliveira43) [inline velero-plugins/pod/restore.go:166]: NIT: this comment would be better to read inside `else`?

**Review by mateusoliveira43 — APPROVED** [review]

**Review by sseago — APPROVED** [review]

**Comment by shubham-pampattiwar** [comment]
Thanks @openshift-cherrypick-robot

**Pull request merged**